services/ansibler/server/server.go

```diff
@@ -139,13 +142,15 @@ func (*server) SetUpLoadbalancers(_ context.Context, req *pb.SetUpLBRequest) (*p
 		lbInfos         = make(map[string]*LBInfo)        //[k8sClusterName]lbInfo
 		k8sNodepools    = make(map[string][]*pb.NodePool) //[k8sClusterName][]nodepools
 		k8sNodepoolsKey = make(map[string]string)         //[k8sClusterName]keys
+		clusterIDs      = make(map[string]string)         //[k8sClusterName]prefix (clusterName + hash)
```

Comment:
Just for the consistency
`//[k8sClusterName]prefix (clusterName + hash)`  -> `//[k8sClusterName]clusterID (<clusterName>-<hash>)`